TheMovieDb/Views/MovieCell.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  MovieCell.swift
+//  TheMovieDb
+//
+//  Created by Rob Cruz on 07/11/21.
+//
+
+import UIKit
+import Kingfisher
+
+class MovieCell: UITableViewCell {
+
+    @IBOutlet weak var movieImageView: UIImageView!
+    @IBOutlet weak var ratingLabel: UILabel!
+    @IBOutlet weak var releaseDateLabel: UILabel!
+    @IBOutlet weak var movieOverviewLabel: UILabel!
+    @IBOutlet weak var movieTitleLabel: UILabel!
+    
+    private var urlString: String = ""
+    
+    func setCellWithValuesOf(_ movie: Movie) {
+        updateUI(title: movie.title, releaseDate: movie.releaseDate, rating: movie.voteAverage, overview: movie.overview, poster: movie.posterPath)
+    }
+    
+    // MARK: - Populate Cells with data
+    
+    private func updateUI(title: String?, releaseDate: String?, rating: Double?, overview: String?, poster: String?) {
+        self.movieTitleLabel.text = title
+        self.movieTitleLabel.adjustsFontSizeToFitWidth = true
+
+        self.releaseDateLabel.text = releaseDate?.readableDate()
+        guard let average = rating else { return }
```

Comment:
Premature returning for one property guard, insinuates average is a requirement for method to assign overview text.

Reply:
A good indication of wrongful use of guard is when there are not exclusively used at the top of the scope.